PIL/Image.py

```diff
@@ -2531,6 +2543,33 @@ def registered_extensions():
         init()
     return EXTENSION
 
+def register_decoder(name, decoder):
+    """
+    Registers an image decoder.  This function should not be
+    used in application code.
```

Comment:
If this function shouldn't be used in application code, should it begin with an underscore?